INSTALL.md

```diff
@@ -40,6 +40,13 @@ $> git clone https://github.com/savoirfairelinux/sous-chef
 $> docker-compose build
 $> docker-compose up
```

Comment:
If you are OK with this, please update the instructions to mention running [Docker for Mac beta](https://docs.docker.com/docker-for-mac/).